mmdet/datasets/pipelines/transforms.py

```diff
@@ -1530,3 +1530,67 @@ def __repr__(self):
         repr_str += f'test_mode={self.test_mode}, '
         repr_str += f'test_pad_mode={self.test_pad_mode})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class CutOut(object):
+    """Randomly drop some regions of image.
+
+    Args:
+        n_holes (int | tuple[int, int]): Number of regions to be dropped.
+            If it is given as a list, number of n_holes will be randomly
+            selected from `n_holes[0]` to `n_holes[1]`.
+        cutout_shape (tuple[int, int] | list[tuple[int, int]] |
+            tuple[float, float] | list[tuple[float, float]]): The candidate
+            shape of dropped regions. It can be
+                1. `tuple[int, int]`: use a fixed cutout shape.
+                2. `tuple[float, float]`: use a fixed ratio of image shape.
```

Comment:
I suggest adding an argument `cutout_ratio` for this case. `cutout_shape` and `cutout_ratio` cannot be both specified.

Reply:
I agree adding another `cutout_ratio` argument since `cutout_shape` now is too flexible and may be confusing.